tools/depends/configure.ac

```diff
@@ -89,6 +89,12 @@ AC_ARG_ENABLE([gplv3],
   [use_gplv3=$enableval],
   [use_gplv3=yes])
 
+AC_ARG_ENABLE([wayland],
+  [AS_HELP_STRING([--enable-wayland],
+  [build libraries needed for Wayland. default is no])],
+  [enable_wayland=$enableval],
+  [enable_wayland=no])
```

Comment:
IMO there should be a check if wayland can be used for the target platform.
OSX, ios, and android can't use it